src/main/resources/view/McTheme2.css

```diff
@@ -0,0 +1,393 @@
+.background {
```

Comment:
Is this file the alternative color scheme? Maybe can make use of `UserPrefs` to allow users to switch to this. Also would be good if the duplicates in the two themes are condensed into one place so that we don't need to edit both when including new GUI elements. Otherwise there will be two representations of some parts of our GUI styling.